components/bases-locales/charte/partner.js

```diff
@@ -1,11 +1,10 @@
-
 import {useState} from 'react'
 import Image from 'next/image'
 import PropTypes from 'prop-types'
-
-import colors from '@/styles/colors'
 import {ChevronDown, ChevronUp} from 'react-feather'
 
+import theme from '@/styles/theme'
```

Comment:
Pour éviter de répéter `theme.colors`, tu peux aussi importer de cette façon.
```suggestion
import theme, {colors} from '@/styles/theme'
```